manifests/init.pp

```diff
@@ -89,6 +91,9 @@
   if $exec_restorecon {
     create_resources ( 'selinux::exec_restorecon', $exec_restorecon )
   }
+  if $login {
```

Comment:
while this style is consistent with the rest of the class, it doesn't meet best practices. using create_resources is deprecated. Instead I sugges to default $login to an empty hash and iterate on it

```puppet
$login.each |$login_name, $login_attributes {
  selinux::login { $login_name:
    * => $login_attributes,
  }
}
```

Reply:
you can drop the `if $login`. `$login.each` will notice that $login is empty and not iterate on it.